### DISCUSSION TITLE
PIC32MM progress

### DISCUSSION THREAD

**Discussion by cb88** [chipKIT32/chipKIT-core#467] [Ideas]
If anyone still wants one I can host or mail out the PIC32MM board design I have. It's a basic breakout board for a PIC32MM0256GPM048. I think I already sent Matt and one other I want to say went to Brian but it has been about 2 years since I messed with it. I finally got around to updating the prototype to fix the regulator circuit I swapped it to a SOT-223 version of the LD1117S33TR regulator while I was at it. I'll probably swap the power selection jumper for a swtich before I send it off also.

![image](https://user-images.githubusercontent.com/1026000/117331003-4dca5380-ae64-11eb-896a-f13767ac0d86.png)

**Comment by JuanDValenciano** [comment]
the idea is to make a board with the same form factor of the arduino auxiliary boards.

How are you going with the development?

**Reply by cb88** [reply]
Not sure just what you mean this board was intended just to break out a pic32mm (without the awkward Arduino form factor as such it has properly aligned headers). Hardware wise it's done. I am not aware of any progress in development other than the occasional discussion of issues with PIC32mm that have occurred. The compiler supports it I believe but the linker scripts and bootloaders aren't ready. also this is a 50x50mm board its a bit smaller than an arduino.

**Reply by majenkotech** [reply]
We need linker scripts, board definitions, boot loader and most importantly core support for the internal peripherals and interfaces. Plus we're really wanting the new compiler, which is a long way off yet as I am only one man.

**Comment by JacobChrist** [comment]
I am all of a sudden very interested in this board.  I would happily pay for the parts and shipping to get one, especially if you have linker scripts working with chipKIT.

Jacob

**Reply by majenkotech** [reply]
We need more than just linker scripts. The core support is going to be the hardest - or is that what yow want to work on with this board...?

**Reply by JacobChrist** [reply]
I was hoping that all that stuff was in place, but if its not, I may have to stick with the MX chips for my current project.  The mm chips look really appealing though.

**Reply by cb88** [reply]
This board is just hardware, its been tested with a basic blink program using the x32 compiler and toolchain... and a pickit3 with IPE.

**Reply by JacobChrist** [reply]
Yeah, like I said above, I think I'll stick with chips that are closer to the Fubarino pedigree for the current project so as to not extend the development.

Thanks,

Jacob
